### ISSUE TITLE
Documentation loop for DO_SET_ROI commands

### ISSUE THREAD

**Issue by AndKe** [mavlink/mavlink#2061]
![image](https://github.com/mavlink/mavlink/assets/1468842/c04a18cd-759b-4366-b8ad-397e81e6ade3)

https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ROI

Please observe that the command is deprecated, and replaced by itself.
I was checking if the altitude parameter could be "-1" - to be ignored so that only yaw was affected by ROI - please follow the blue text on parameters 5,6 or 7   - it does not provide any additional information.

**Comment by hamishwillee** [comment]
No, this says `DEPRECATED: Replaced by MAV_CMD_DO_SET_ROI_*`. That means messages that start with `MAV_CMD_DO_SET_ROI_`. So MAV_CMD_DO_SET_ROI_LOCATION, MAV_CMD_DO_SET_ROI_WPNEXT_OFFSET, and so on.

Would it help to change this to: `DEPRECATED: Replaced by separate messages starting with MAV_CMD_DO_SET_ROI_, such as MAV_CMD_DO_SET_ROI_LOCATION`.  ?

W.r.t. to the values, I'd interpret them in terms of the new messages - e.g. for WPNEXT see what the param 7 values says for https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ROI_WPNEXT_OFFSET

Does that answer?

PS Sorry for delay, only work Weds, Thurs.

**Comment by hamishwillee** [comment]
This is OK. Closing.
